src/connector/src/sink/iceberg/mod.rs

```diff
@@ -370,6 +370,9 @@ impl Sink for IcebergSink {
     const SINK_NAME: &'static str = ICEBERG_SINK;
 
     async fn validate(&self) -> Result<()> {
+        if "snowflake".eq_ignore_ascii_case(self.config.catalog_type()) {
```

Comment:
Do we need to get a licence-key to use Snowflake sink?

Reply:
cc @xxhZs @chenzl25

Reply:
IIUC,snowflake catalog sink isn't supported.